05/app.js

```diff
@@ -34,4 +34,35 @@ function insertUsers(usersList) {
 
         ulElement.appendChild(liElement);
     });
+    getNewUser(usersList.length);
 }
+
+function getNewUser(id) {
```

Comment:
Raczej użył bym nazwy `saveNewUser` lub `addUser` - to bardziej sugeruje co się dzieje.